raiden/channel.py

```diff
@@ -599,8 +600,8 @@ def channel_closed(self, block_number):
     def blockalarm_for_settle(self, block_number):
         def _settle():
             for _ in range(3):
-                # do not call settle if already settled, the event polling
-                # might be lagging behind.
+                # do not call settle if already settled, the settle_event might
+                # not be set if the LogListener is lagging behind.
```

Comment:
argh that sucks! That's why you had so many commits in other times. Good to see you know it now. Generally a lot of small commits is nice but not too many. If you just append something to a previous commit you should generally amend it and force push (only if it's your own feature branch ofcourse)